### ISSUE TITLE
Discussion: Determining what files are installed for Thor "Apps"

### ISSUE THREAD

**Issue by Jimrnelson** [VFPX/VFPXDeployment#21]
Thor "Apps" are loosely defined as those projects which create Thor tools (as distinct from "Components", which are expected to be incorporated into production apps.)

Before the transition to GitHub, Thor CFU provided the simplest way to download the source code for "Apps" and "Components", so that using CFU always downloaded the entire source code.

Now, however, for "Apps" at least, I think this no longer makes sense.  Thor now provides easy access to the home page for all projects:

![image](https://github.com/VFPX/VFPXDeployment/assets/81179335/c420190b-04f3-4d22-84a6-731921099fed)

and users can download the source code from there.

My suggestion for "Apps" is that the "Installed Files" are only those files needed to actually run the app.  Thus, for many, a single file.  GoFish.App is (I believe) completely stand alone.

This should remove duplication in our forked VFPX projects and have other benefits as well.

What's everybody's thought on this?

**Comment by DougHennig** [comment]
I agree. I made that point in my VFF "Anatomy of a VFPX Project" session. And in a future release of Project Explorer, source code will be excluded.
